samples/dfu/README.rst

```diff
@@ -108,12 +108,9 @@ Build Zephyr sample application for nRF9160 DK:
    $ west build -b nrf9160dk_nrf9160_ns samples/dfu
    $ west flash
 
-Now rebuild the application with the new version number 1.2.3 to distinguish it
-from the old firmware:
-
-.. code-block:: console
-
-   $ west build -b nrf9160dk_nrf9160_ns samples/dfu -- -DCONFIG_MCUBOOT_IMAGE_VERSION=\"1.2.3\"
```

Comment:
We should still be able to override the `CONFIG_GOLIOTH_SAMPLE_FW_VERSION` symbol from the `west build` command right? I don't think we need to document it as part of the sample, but that could be useful for clients (for example in CI systems where the version is automatically incremented), so just checking that the functionality remains.